src/style.css

```diff
@@ -0,0 +1,134 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  display: block;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--main-text-color);
+  font-style: normal;
+}
+
+.card__code {
+  display: block;
+  /*font-style: normal;
+  font-weight: 400;*/
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--second-text-color);
+  margin-bottom: 16px;
+}
+
+.card__raiting {
+  display: flex;
+  text-align: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-text-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-price {
+  color: var(--second-text-color);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-cost {
+  color: var(--main-text-color);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buttom {
+  display: block;
+  text-align: center;
+  background: #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  line-height: 40px;
+  height: 40px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__buttom:hover {
+  border: 1px solid var(--button-color);
+  background-color: #fff;
+  color: var(--button-color);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
move font-family: "Roboto", Arial, sans-serif; to html tag since only box-sizing: border-box; is allowed to use at * selector